stable/jenkins/master-image/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM jenkinsci/jenkins:2.46.1
+FROM jenkinsci/jenkins:2.53
```

Comment:
Generally speaking no one runs the latest weekly release in production and IMO this chart should sticky to the latest lts release which at the the of this comment is 2.46.1. 

https://jenkins.io/changelog-stable/